frontend/src/integrations/groupsio/components/groupsio-connect-drawer.vue

```diff
@@ -0,0 +1,379 @@
+            required: 'This field is required',
+            url: 'Enter valid email',
+          }"
+        >
+          <!-- <div class="text-2xs text-gray-500 leading-normal mb-1">
+            Hostname of your community instance in Discourse.
+          </div> -->
+          <el-input ref="focus" v-model="form.email" type="email" @blur="onBlurEmail()" />
+        </app-form-item>
+        <app-form-item
+          v-if="!isAPIConnectionValid"
+          class="mb-6"
+          :validation="$v.apiKey"
+          label="Password"
+          :required="true"
+          :error-messages="{
+            required: 'This field is required',
+          }"
+        >
+          <el-input ref="focus" v-model="form.password" @blur="onBlurPassword()">
+            <template #suffix>
+              <div
+                v-if="isValidating"
+                v-loading="isValidating"
+                class="flex items-center justify-center w-6 h-6"
+              />
+            </template>
+          </el-input>
+        </app-form-item>
+        <app-form-item
+          v-if="!isAPIConnectionValid"
+          class="mb-6"
+          :validation="$v.twoFactorCode"
+          label="2FA Code (optional)"
+        >
+          <el-input ref="focus" v-model="form.twoFactorCode" @blur="onBlurTwoFactorCode()">
+            <template #suffix>
+              <div
+                v-if="isValidating"
+                v-loading="isValidating"
+                class="flex items-center justify-center w-6 h-6"
+              />
+            </template>
+          </el-input>
+        </app-form-item>
+
+        <div class="flex flex-row gap-2">
+          <el-button
+            v-if="!isAPIConnectionValid"
+            class="btn"
+            :disabled="!isVerificationEnabled"
+            :loading="isVerifyingAccount"
+            @click="validateAccount()"
+          >
+            Verify Account
+          </el-button>
+
+          <el-button v-if="isAPIConnectionValid" class="btn" @click="reverifyAccount()">
+            Reverify Account
+          </el-button>
+
+          <div v-if="accountVerificationFailed" class="mt-1">
+            <i class="ri-error-warning-line text-red-500 w-[14px] mr-2" />
+            <span class="text-red-500 text-[13px]">Authentication failed</span>
+          </div>
+        </div>
+      </el-form>
+      <el-divider />
+      <div class="w-full flex flex-col mb-6" :class="isAPIConnectionValid ? 'opacity-100' : 'opacity-50'">
+        <p class="text-[16px] font-semibold">
+          Connect groups
+        </p>
+        <div class="text-2xs text-gray-500 leading-normal mb-1">
+          Select which groups you want to track. All topics avaliable in each group will be monitored. <a
+            href="https://docs.crowd.dev/docs/discourse-integration#webhooks"
+            target="_blank"
+            rel="noopener noreferrer"
+            class="hover:underline"
+          >Read more</a>
+        </div>
+      </div>
```

Comment:
This section seems disabled and out of place.
I would either:
- If API connection is invalid, don't show this section at all. If valid, show the section
- If API connection is invalid show the entire section, but only the form fields are disabled. If valid, all form fields get enabled. So yeah this text is never at opacity-50

Doesn't make much sense to "disable", make this section transparent, if user cannot see anything apart from the text.

I would go for option 2, since you have valuable information, such as a link in there. Just remove the opacity and disable the fields below 😄 

<img width="580" alt="Screenshot 2023-09-01 at 17 10 24" src="https://github.com/CrowdDotDev/crowd.dev/assets/20134207/dbc10b6b-6ef5-42bf-9512-4946eca8057e">

Reply:
Removed capacity and disabled fields when needed